tensordict/persistent.py

```diff
@@ -228,10 +228,7 @@ def close(self):
 
     def _process_key(self, key):
         key = unravel_keys(key)
```

Comment:
one more general thought: shouldn't `unravel_keys` be called `unravel_key`?
cause it always acts on a single key.
especially cause now we have `unravel_key_list` for multiple keys

Reply:
yep I agree :) let me do that